assets/focus-visible.styl

```diff
@@ -1,3 +1,3 @@
 // https://github.com/WICG/focus-visible#2-update-your-css
-.js-focus-visible :focus:not(.focus-visible)
+.js-focus-visible :focus:not(.focus-visible, [data-focus-visible-added])
```

Comment:
Looks like [data-focus-visible-added] was added in https://github.com/WICG/focus-visible/pull/219.  Their current docs have this example:

![image](https://user-images.githubusercontent.com/77567/145478108-8bbaf786-a53e-469c-88f9-dcdefde9a0da.png)

Maybe we just update this to their alternate approach, so like:

```suggestion
[data-js-focus-visible] :focus:not([data-focus-visible-added])
```

What do you think?